GameData/RP-1/Contracts/Earth Observation 1/EOSCorona.cfg

```diff
@@ -0,0 +1,141 @@
+
+	synopsis = Launch the first dedicated imagery satellite.
+
+	completedMessage = Well done! Never before has so much of the Earth been photographed at one time.  This new capability will change the way the world is viewed forever.
+
+	sortKey = 402
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 80
+	failureReputation = 0 // was @rewardReputation
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarlyEOS
+	}
+
+	REQUIREMENT	
+	{	name = CompleteContract
+		type = CompleteContract
+		contractType = first_OrbitRecover
+		title = Completed First Orbit Recovery
+	}
+
+	PARAMETER
+	{
+		name = Corona1
+		type = VesselParameterGroup
+		title = First high resolution imagery satellite
+		define = EOSCorona
+
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a new vessel
+			hideChildren = true
+		}
+		PARAMETER 
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 0
+			maxCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+
+		PARAMETER
+		{
+			name = OrbitSequence
+			type = Sequence
+			title = Collect orbital imagery for 2 days and return it from orbit
+
+			PARAMETER
+			{
+				name = HasPower
+				type = HasResource
+				resource = ElectricCharge
+				minQuantity = 1.0
+				title = Craft must have Electric Charge for two days
+				hideChildren = true
+			}
+
+			PARAMETER
+			{
+				name = PartValidation
+				type = PartValidation
+				
+				VALIDATE
+				{
+					MODULE
+					{
+						name = Experiment
+						experimentID = RP0photos2
+					}
+				}
+				
+				title = Has orbital cameras
+			}
+			
+			PARAMETER
+			{
+				name = Orbit
+				type = Orbit
+				maxApA = 200000
+				maxEccentricity = 0.035
+				minInclination = 80
+				maxInclination = 100
+				disableOnStateChange = true
+				title = Achieve a polar orbit no higher than 200km with less than 0.035 eccentricity
+			}
+
+			PARAMETER
+			{
+				name = Duration
+				type = Duration
+				duration = 2d
+				preWaitText = Remain in orbit
+				waitingText = Exposing film...
+				completionText = Orbit period complete
+			}
+
+			PARAMETER
+			{
+				name = ReturnHome
+				type = RP1ReturnHome
+				title = Return the film to the surface
+				hideChildren = true
+				completeInSequence = true
+			}		
+		}

```

Comment:
This doesn't have a science parameter--I assume the issue is the same as the EOSCoronaRep that you mentioned?

Reply:
Correct, have been holding off on those 2 until I knew how to do the parameter